client/src/pages/GalleryPage.js

```diff
@@ -1,10 +1,30 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components/macro";
 import { useEffect } from "react";
 import { getImageObj } from "../utils/api";
 import useAsync from "../utils/useAsync";
+import { Button } from "../components/Button";
 
-const Display = styled.div`
+const ImageDisplay = styled.div`
+  border: solid 1px lightgray;
+  display: flex;
+  justify-content: center;
+`;
+// const Display = styled.div`
+//   border: solid 1px lightgray;
+//   display: flex;
+//   justify-content: center;
+// `;
+const ImageContainer = styled.div`
+  height: 800px;
+  width: 800px;
```

Comment:
Are you sure these fixed sizes are necessary? I would expect the parent element to have a size, not determined by its content, but rather a flexible size, which is then passed to its child elements.

Also looking at the implementation, this container has several child elements, an image and two buttons. Wouldn't it make sense to have this as the flexbox and replace `ImageDisplay`?

Reply:
Thanks! I'm still working on a good solution for the image display.